day8/alienalphabet.py

```diff
@@ -0,0 +1,127 @@
+##############
+## Solution ##
+##############
+
+def alienAlphabet(orderedWords):
+    # edge cases
+    if len(orderedWords) <= 0:
+        return ''
+    if len(orderedWords) == 1:
+        return removeDups(orderedWords[0])
```

Comment:
I think I should have added more explanation to what I was doing here. @jabagawee is correct about my decision to emit an arbitrary ordering, and I wanted to default to the order the letters appeared in originally, although I see that doing that actually added a lot of overhead that I don't need at this point.